### PR TITLE
Add slate docs for new ip restriction error code

### PR DESCRIPTION
**Pull request by mykeoye**
### Context
This pull request adds documentation for a new error code which indicates that an ip address is restricted from accessing a resource

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -154,6 +154,7 @@ error categories, the last two digits define specific errors.
 * 0209: Two Factor authentication required
 * 0210: Device authentication required
 * 0211: Device authentication failed (Invalid IP or expired Token)
+* 0212: IP address is not allowed to access this resource
```
Comment (thenewversion) [inline source/index.html.md:157]: 0212 is already used by Quoterbot :/

Comment (mykeoye) [reply source/index.html.md:157]: Thanks @thenewversion please do you know what message it returns so i document while i take the next available code `0213` ?

Comment (thenewversion) [reply source/index.html.md:157]: It simply returns access denied but is very specific for Quoterbot so it should not really be added to that list unless marked as `reserved for internal use`

Comment (mykeoye) [reply source/index.html.md]: Okay thanks

File: source/index.html.md [hunk 2]
```diff
@@ -154,6 +154,7 @@ error categories, the last two digits define specific errors.
 * 0209: Two Factor authentication required
 * 0210: Device authentication required
 * 0211: Device authentication failed (Invalid IP or expired Token)
+* 0213: IP address is not allowed to access this resource
```
Comment (viccomx) [inline source/index.html.md:157]: Please add what @thenewversion asked for.

Comment (mykeoye) [reply source/index.html.md:158]: Updated @viccomx

**Review by viccomx — APPROVED** [review]
LGTM

**Pull request merged**